llama-index-integrations/vector_stores/llama-index-vector-stores-kdbai/llama_index/vector_stores/kdbai/utils.py

```diff
@@ -0,0 +1,19 @@
+def default_sparse_encoder(texts: list[str]) -> list[dict[int, int]]:
```

Comment:
typing should use the `from typing import List, Dict, ..` types, to work with python3.8

Reply:
Updated